src/lib/xmlparse.rs

```diff
@@ -336,6 +338,9 @@ struct CXmlHandlers<'scf> {
     m_startDoctypeDeclHandler: XML_StartDoctypeDeclHandler,
     m_startElementHandler: XML_StartElementHandler,
     m_startNamespaceDeclHandler: XML_StartNamespaceDeclHandler,
+    m_unknownEncoding: Option<Box<UnknownEncoding>>,
```

Comment:
Those fields seemed like they related to the handler info, and I wanted to make `handleUnknownEncoding` a handler method since it does a bunch of handler stuff involving those fields. It may or may not have been related to borrowing errors, I forget.

Reply:
Those fields seemed like they related to the handler info, and I wanted to make `handleUnknownEncoding` a handler method since it does a bunch of handler stuff involving those fields.

Reply:
Sounds good, it's fine.